mslib/msui/mss_pyui.py

```diff
@@ -416,6 +418,15 @@ def closeEvent(self, event):
                 self.mscolab_window.close()
             if self.config_editor is not None:
                 self.config_editor.close()
+            """
+            tv = tableview ; it is an object of MSS Table View Window
+            Tableview was not closing even when main window closes
+            But after this, it closes fine.
+            """
+            if self.tv is not None:
+                if self.tv.flag_tv() == 0:  # checks whether or not tableview window has closed before
```

Comment:
see comments below, a better name for this and I think we can use a boolean.

Reply:
Yeah, we can use a boolean. No problem in that. Also, tv_window_exists sounds like a better name for this boolean behaviour. I have noted all points. I will send a PR to stable. Thanks.